autoload/go/alternate.vim

```diff
@@ -0,0 +1,36 @@
+" By default use edit to switch to the new alternate buffer
+if !exists("g:go_alternate_mode")
+	let g:go_alternate_mode = "edit"
+endif
+
+" By default create the test file if it does not exist
+if !exists("g:go_create_test_file")
+	let g:go_create_test_file  = 1
```

Comment:
Can you disable this? It should give an error if the file doesn't exists. Having it enabled by default will cause to create unwanted files. We should be very simple in the features and not disturb people's worklofws.